### DISCUSSION TITLE
Hola, quisiera consultar como puedo plotear la matriz de confusión

### DISCUSSION THREAD

**Discussion by PabloMoyaGalvez** [dlopezg/mvpalab#11] [General]
Hola David, primero muchas gracias por tu trabajo.
Te consulté ya hace algún tiempo por correo electrónico una dificultad para utilizar mvpalab a lo que me ayudaste y te agradezco nuevamente.
Hoy estoy retomando una etapa de un análisis con mvpalab y quería saber si hay alguna forma de plotear la matriz de confusión directamente con alguna función de mvpalab.
Saludos Atentamente
Pablo

**Comment by dlopezg** [comment]
Buenos días Pablo,

Actualmente MVPAlab no representa gráficamente las matrices de confusión, pero al tratarse de clasificaciones binarias son muy fáciles de ver e interpretar simplemente mostrando los valores de la matriz 2x2. Aún así, Matlab de forma nativa proporciona varias funciones para representar matrices de confusión. Una de ellas es **_confusionchart(C)_**, a la que le puedes pasar como argumento la matriz de confusión C obtenida con mvpalab y te la representa gráficamente. 

Espero haberte ayudado, un saludo.
David

**Reply by PabloMoyaGalvez** [reply]
Hola David, efectivamente no tuve problemas para acceder a la matriz generada por mvpalab muchas gracias por tu ayuda y la pronta respuesta.
